rwengine/src/audio/SoundBuffer.cpp

```diff
@@ -15,6 +15,14 @@ SoundBuffer::SoundBuffer() {
     alCheck(alSourcei(source, AL_LOOPING, AL_FALSE));
 }
 
+SoundBuffer::~SoundBuffer() {
+    if (this->isPlaying()) {
```

Comment:
Here and below `this`I think is redundant.

Reply:
You're right, I can remove the test and stop.
I don't know whether this is defined behavior?